src/App.js

```diff
@@ -1,25 +1,36 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { useEffect, useState } from "react";
+import './App.css';
+import { useEffect, useState } from 'react';
 
 function App() {
-  const [data, setData] = useState(null)
+  const [data, setData] = useState({});
   useEffect(() => {
     async function loadData() {
-      const response = await fetch('https://redi-final-restaurants.herokuapp.com/restaurants')
-      const data = await response.json()
-      setData(data)
+      const response = await fetch(
+        'https://redi-final-restaurants.herokuapp.com/restaurants'
+      );
+      const data = await response.json();
+      setData(data);
     }
-    loadData()
-  }, [])
-  console.log(data)
+    loadData();
```

Comment:
there are a lot of inserted or removed semi colons, you should decide with your team on a code styling and then ensure that everyone sticks to it. That was the code is consistent and and you done have changes on lines that you didnt touch, like this one